airflow/plugins/operators/amplitude_to_flattened_json.py

```diff
@@ -12,24 +15,35 @@
 DATE_FORMAT = "%Y%m%dT%H"
 
 
-def amplitude_to_df(start: str, end: str, api_key: str = None, secret_key: str = None):
```

Comment:
This is a silly linting issue, but it will have issues upstream:

https://github.com/cal-itp/data-infra/pull/1116/files#diff-3444b282154ce84290a73db1baea914cfd37f7737f850ef13457eaf3899aa51fR15

This repo likes the line to be like this: `def amplitude_to_df(start: str, end: str, api_key: str = None, secret_key: str = None):`

https://github.com/cal-itp/data-infra/pull/1116/commits/2da366e7eb454fd2577551647e71f843bc81ce63

Reply:
Thanks for mentioning this @machikoyasuda !

When I re-format this parameter list into one line, it fails the `black` hook when I run `pre-commit run --all-files`. The `black` hook changes the code back to having each parameter on its own line. 😞 (screenshots below)

I am ok with making this one line and seeing what happens when the check is re-run on #1116. Also open to other suggestions!

Screenshots:
![image](https://user-images.githubusercontent.com/25497886/157559599-c02f228f-bc06-4129-8dd7-0fb324524e0c.png)
![image](https://user-images.githubusercontent.com/25497886/157559621-7d2a1280-5a2b-4c62-94e3-2e3306497e8b.png)